extensionsGallery.json

```diff
@@ -527,7 +527,58 @@
                     ],
                     "statistics": [],
                     "flags": "preview"
-                }
+                },
+                {
+                    "extensionId": "22",
+                    "extensionName": "firstresponderkit",
+                    "displayName": "First Responder Kit",
+                    "shortDescription": "Access current versions of scripts from the First Responder Kit",
+                    "publisher": {
+                      "displayName":"Drew Skwiers-Koballa",
+                      "publisherId": "drewsk",
+                      "publisherName":"drewsk"
+                    },
+                    "versions": [
+                      {
+                        "version": "0.1.0",
+                        "lastUpdated": "7/25/2018",
+                        "assetUri": "",
+                        "fallbackAssetUri": "fallbackAssetUri",
+                        "files": [
+                        {
+                          "assetType": "Microsoft.SQLOps.DownloadPage",
+                          "source": "https://github.com/dzsquared/sqlops-firstresponderkit/releases/tag/0.1.0"
+                        },
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.Links.Source",
+                          "source": "https://github.com/dzsquared/sqlops-firstresponderkit"

```

Comment:
please add the source as a property:

"properties": [
                          { "key": "Microsoft.VisualStudio.Code.ExtensionDependencies", "value":""},
                            { "key":  "Microsoft.VisualStudio.Services.Links.Source", "value": ""https://github.com/dzsquared/sqlops-firstresponderkit"},
                         { "key": "Microsoft.VisualStudio.Code.Engine", "value":"*"}
                       ]